Database/Corrections/sodQuestFixes.lua

```diff
@@ -2737,6 +2737,18 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -425762,
             [questKeys.zoneOrSort] = sortKeys.HUNTER,
         },
+        [90170] = {
+            [questKeys.name] = "Flanking Strike",
+            [questKeys.startedBy] = {{208124,3099,3100,3225}},
```

Comment:
We have other quests like this as well, with the "kill any of this very unspecific mob across the zone and then go to this one place" (the other "Flanking Strike" quests above as example).

I am wondering if it is the better approach to only show the location where you need to go at the end and remove the unspecific mobs from the `startedBy`.

In this example we would only leave 208124 in, remove the rest and improve the `objectivesText` to state that you need to kill boars in Durotar 🤷🏻

Reply:
This may be a better option, however there are other quests that are just different locations for 'Flanking Strike', that have 6-7 mobs, and so listing all of those out in the objectivesText may be a bit of a pain.

Reply:
Yeah, maybe those objectives could be better in the wording like "acquire item X which you get from A, B and C for example" and not list ALL locations, only the "best".

Reply:
Okay, a cursory solution is here: https://github.com/Questie/Questie/pull/5518/commits/6557e6b64b1ea11e8dac578879cd27c6eb14daad

As all of the NPCs required have the suffix 'Mottled Boar' I think this is the elegant way of doing it.

Reply:
Beautiful 🥳

Reply:
I'll go through the already added runes on `master` and see if I can do something similar for them